website/docs/execution-node/fee-recipient.md

```diff
@@ -1,83 +1,192 @@
-`fee-recipient-config-url` flag requires this json format provided through url.
+#### Fee Recipient JSON Config File
 
-This will allow you to map your validators to corresponding eth addresses or generally cover the remaining keys with a default.
+If you use either `fee-recipient-config-file` or `fee-recipent-config-url` to specify your fee recipient address, your JSON configuration should follow this schema:
 
 ```
 {
   "proposer_config": {
     "0xa057816155ad77931185101128655c0191bd0214c201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a": {
       "fee_recipient": "0x50155530FCE8a85ec7055A5F8b2bE214B3DaeFd3",
+    },
+    "0xac201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a057816155ad77931185101128655c0191bd0214": {
+      "fee_recipient": "0x5A5F8b2bE214B3DaeFd350155530FCE8a85ec705",
     }
   },
   "default_config": {
     "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A"
   }
 }
 ```
-- `proposer_config` is optional
-- `default_config` is mandatory
-- `fee_recipient` is mandatory in each config
+<br />
+
+The above JSON demonstrates configuring two 1:1 mappings between `validator public key`:`fee_recipient` and a default `fee_recipient`. In this case, the `default_config` fee recipient address would apply to all validator public keys not specified in `proposer_config`. JSON configuration members are listed in the following table:
+
+<br />
+
+<table>
+  <tr>
+    <th>Member</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>proposer_config</code></td>
+    <td>
+    Optional. Your validator client instance’s public key. <br /> <br /> 
+    <strong>Type:</strong> Validator public key. 98 characters long.<br /> <br /> 
+    <strong>Note:</strong> Use this to map a single validator instance public key to a single fee recipient ETH address.
+    </td>
+  </tr>
+  <tr>
+    <td><code>proposer_config.fee_recipient</code></td>
+    <td>
+    Optional. Required if validator instance public key is provided via <code>proposer_config</code>.  <br /> <br /> 
+    <strong>Type:</strong> ETH address. 42 characters long. <br /> <br /> 
+    <strong>Note:</strong> Use this to map a single validator key to a single ETH fee recipient address.
+    </td>
+  </tr>
+  <tr>
+    <td><code>default_config</code></td>
+    <td>
+    Required. 
+    </td>
+  </tr>
+  <tr>
+    <td><code>default_config.fee_recipient</code></td>
+    <td>
+    Required.<br /> <br /> 
+    <strong>Type:</strong> ETH address. 42 characters long. <br /> <br /> 
+    <strong>Note:</strong> This sets the default ETH address for all remaining validator public keys that don’t have 1:1 mapping already from the <code>proposer_config</code> member.
+    </td>
+  </tr>
+</table>
+
+<br />
+
+
+### Configuring Fee Recipient on your beacon node
+
+A fee recipient address can be configured on your beacon node instance by using the `suggested-fee-recipient` flag.
+
+:::warning This is an additive flag
```

Comment:
all flags are additive we probably don't need this.

Reply:
Removed.